source/_integrations/nissan_leaf.markdown

```diff
@@ -113,6 +113,7 @@ You can also use the `nissan_leaf.update` service to request an on-demand update
 
 ## Hints
 
+* The `update_interval` is the time **between** updates. A high update interval will refresh rarely, and a low one refreshes more frequently. 
```

Comment:
This is already described on line 47.

Reply:
I know. That's how I was able to guide the two people that were asking on social whether a low update interval would exascerbate or reduce the 12V load. If there's ambiguity, I'd seek to reduce it, but I appreciate a sample size of two probably isn't that significant.  Will defer to your preference  😃

Reply:
We should not add another line.
If line 48 needs adjusting, please adjust that instead.

Reply:
Believe that's now done.